### ISSUE TITLE
 JNI DETECTED ERROR IN APPLICATION: illegal class name 'com.openmmlab.mmdeployxdetector.MMDeployDetector'

### ISSUE THREAD

**Issue by dhzgit** [hanrui1sensetime/MMDeployX-prototype#1]
直接运行时会提示找不到libjavaapis.so文件  修改文件到jniLibs后可以找到so文件 但是任然报错 会提示

 JNI DETECTED ERROR IN APPLICATION: illegal class name 'com.openmmlab.mmdeployxdetector.MMDeployDetector'

![image](https://user-images.githubusercontent.com/49825458/176862735-acc2f29a-ba59-4b1e-bc8d-8997e4f3418d.png)

**Comment by hanrui1sensetime** [comment]
#2 is working in progress. This bug will fixed after #2 complete.

**Comment by hanrui1sensetime** [comment]
#2 is now merged into master.
